contracts/staking/compound/GoodCompoundStaking.sol

```diff
@@ -210,4 +259,20 @@ contract GoodCompoundStaking is SimpleStaking {
 				: ((_amount / (10**decimalDifference)) * er) / 10**mantissa; // calculation based on https://compound.finance/docs#protocol-math
 		return tokenWorth;
 	}
+
+	/**
+	 * @dev Set Gas cost to interest collection for this contract
+	 * @param _amount Gas cost to collect interest
+	 */
+	function setcollectInterestGasCost(uint32 _amount) external {
+		_onlyAvatar();
+		collectInterestGasCost = _amount;
+	}
+
+	function getCompValueInUSD(uint256 _amount) public view returns (uint256) {
+		AggregatorV3Interface tokenPriceOracle =
+			AggregatorV3Interface(nameService.getAddress("COMP_USD_ORACLE"));
```

Comment:
on second thought this shouldnt be in nameservice, it is specific to GoodCompoundStaking

Reply:
changed it